packages/terra-image/src/Image.jsx

```diff
@@ -59,12 +69,13 @@ const Image = ({
       alt={alt}
       height={height}
       width={width}
+      onLoad={onLoad}
+      onError={onError}
```

Comment:
I think the main concern is that if you pass onLoad and onError into the object without them being first class, then they would get added to the image anyway through customProps.